<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5GN5U49" failures="2" tests="2" name="firefoxTest3" time="30.654" errors="0" timestamp="2021-08-03T21:08:01 IST">
  <testcase classname="myDemoprojectMaven.test.ActitimetestM" name="Test1" time="3.35">
    <failure type="java.lang.AssertionError" message="User Test fails due to url not match expected [http://desktop-5gn5u49/reports/reports.do] but found [http://desktop-5gn5u49/user/submit_tt.do]">
      <![CDATA[java.lang.AssertionError: User Test fails due to url not match expected [http://desktop-5gn5u49/reports/reports.do] but found [http://desktop-5gn5u49/user/submit_tt.do]
at myDemoprojectMaven.test.ActitimetestM.Test1(ActitimetestM.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="myDemoprojectMaven.test.ActitimetestM" name="Test2" time="3.143">
    <failure type="java.lang.AssertionError" message="User Test fails due to url not match expected [http://desktop-5gn5u49/administration/userlist.do] but found [http://desktop-5gn5u49/user/submit_tt.do]">
      <![CDATA[java.lang.AssertionError: User Test fails due to url not match expected [http://desktop-5gn5u49/administration/userlist.do] but found [http://desktop-5gn5u49/user/submit_tt.do]
at myDemoprojectMaven.test.ActitimetestM.Test2(ActitimetestM.java:84)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- firefoxTest3 -->
